example/credentials.example.yaml

```diff
@@ -1,3 +1,4 @@
 app-id: <Your app ID>
 app-key: <Your app key>
-app-secret: <Your app secret>
\ No newline at end of file
+app-secret: <Your app secret>
+app-cluster: <Your app cluster. Omit for default>
```

Comment:
Yeah, lots of libraries do assume `mt1` if no cluster is declared and we certainly don't want to change that behaviour as it would break lots of stuff. I think it's just easier if we don't encourage that pattern. As you say, explicitness is preferable 👍